mmv1/products/edgecontainer/Cluster.yaml

```diff
@@ -0,0 +1,515 @@
+      Fleets are a Google Cloud concept for logically organizing clusters,
+      letting you use and manage multi-cluster capabilities and apply
+      consistent policies across your systems.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "project"
+        immutable: true
+        description: |
+          The name of the Fleet host project where this cluster will be registered.
+          Project names are formatted as
+          `projects/<project-number>`.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: "membership"
+        default_from_api: true
+        immutable: true
+        description: |
+          The name of the managed Hub Membership resource associated to this cluster.
+          Membership names are formatted as
+          `projects/<project-number>/locations/global/membership/<cluster-id>`.
+  - !ruby/object:Api::Type::NestedObject
+    name: "networking"
+    required: true
+    description: |
+      Fleet related configuration.
+      Fleets are a Google Cloud concept for logically organizing clusters,
+      letting you use and manage multi-cluster capabilities and apply
+      consistent policies across your systems.
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: "clusterIpv4CidrBlocks"
+        item_type: Api::Type::String
+        immutable: true
+        required: true
+        description: |
+          All pods in the cluster are assigned an RFC1918 IPv4 address from these
+          blocks. Only a single block is supported. This field cannot be changed
+          after creation.
+      - !ruby/object:Api::Type::Array
+        name: "servicesIpv4CidrBlocks"
+        item_type: Api::Type::String
+        immutable: true
+        required: true
+        description: |
+          All services in the cluster are assigned an RFC1918 IPv4 address from these
+          blocks. Only a single block is supported. This field cannot be changed
+          after creation.
+      - !ruby/object:Api::Type::Array
+        name: "clusterIpv6CidrBlocks"
+        item_type: Api::Type::String
+        immutable: true
+        description: |
+          If specified, dual stack mode is enabled and all pods in the cluster are
+          assigned an IPv6 address from these blocks alongside from an IPv4
+          address. Only a single block is supported. This field cannot be changed
+          after creation.
+      - !ruby/object:Api::Type::Array
+        name: "servicesIpv6CidrBlocks"
+        item_type: Api::Type::String
+        immutable: true
+        description: |
+          If specified, dual stack mode is enabled and all services in the cluster are
+          assigned an IPv6 address from these blocks alongside from an IPv4
+          address. Only a single block is supported. This field cannot be changed
+          after creation.
+      - !ruby/object:Api::Type::Enum
+        name: "networkType"
+        default_from_api: true
+        immutable: true
```

Comment:
I don't see this field in the API docs, but it seems like something that should also be `output: true`. Can this be set to contradict the CIDR block configs?

Also, you probably don't need `NETWORK_TYPE_UNSPECIFIED` as an option, and if this is `output: true`, it can probably be changed to a string.

Reply:
Thanks for capturing, this field is output only. It is not in the public doc because currently we mark the field as hidden. 
Also change the field type from Enum -> String type.